src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -268,7 +268,7 @@ const SnapshotListModal = () => {
                         {!checkedSnapshots.size || !rbac?.repoWrite
                           ? 'Remove selected snapshots'
                           : checkedSnapshots.size == count
-                            ? 'Can\'t remove all snapshots'
+                            ? "Can't remove all snapshots"
```

Comment:
changes in this file triggered a couple warnings, maybe we should change the contraction to `cannot`? or we could do what you've done in this [PR](https://github.com/content-services/content-sources-frontend/pull/397/files)?

Reply:
hmm this is eslint and prettier fighting each other. I should likely solve this. 🤔

Reply:
seeing the warnings even with the single quotes 😕 but i think these can be fixed by changing `can't` to `cannot`